### ISSUE TITLE
PPM Receiver

### ISSUE THREAD

**Issue by tjgreystoke** [mirkix/ardupilotblue#15]
Hi @mirkix ,

I am trying to use a PPM receiver - more exactly the FS-IA6B -  with my bbblue copter. I have followed all the software setup instructions and I am able to start the copter with GPS enabled, and connect with the Mission Planner using: 
`sudo ./arducopter-3_5_4 -C tcp:xxx.xxx.xxx.xxx:14550 -B /dev/ttyS2` and my personal Wi-Fi IP.

This is what I am doing to connect the receiver:
1. Use E4 pin 4 to and the 5V power out connector:

2. Use DSM2 pin 3 and the 5V power out connector:
In this configuration I played with the copter switches, as shown below, without any result:
(a). `sudo ./arducopter-3_5_4 -C tcp:xxx.xxx.xxx.xxx:14550 -B /dev/ttyS2 -A /dev/ttyS4`
(b). `sudo ./arducopter-3_5_4 -A tcp:xxx.xxx.xxx.xxx:14550 -B /dev/ttyS2 -C /dev/ttyS4`

In the Mission Planner remote calibration I am not able to see any green bars, and at this point I am out of ideas and looking for help.

Thank you!!

PS I did bind the remote and the receiver.
PPS Although I have basic knowledge of both, I am not a computer scientist nor an electrical engineer, so instructions in a "for dummies" format are appreciated.

**Comment by tjgreystoke** [comment]
Hi @mirkix ,

I am trying to use a PPM receiver - more exactly the FS-IA6B -  with my bbblue copter. I have followed all the software setup instructions and I am able to start the copter with GPS enabled, and connect with the Mission Planner using: 
`sudo ./arducopter-3_5_4 -C tcp:xxx.xxx.xxx.xxx:14550 -B /dev/ttyS2` and my personal Wi-Fi IP.

This is what I am doing to connect the receiver:
1. Use E4 pin 4 to and the 5V power out connector:

2. Use DSM2 pin 3 and the 5V power out connector:
In this configuration I played with the copter switches, as shown below, without any result:
(a). `sudo ./arducopter-3_5_4 -C tcp:xxx.xxx.xxx.xxx:14550 -B /dev/ttyS2 -A /dev/ttyS4`
(b). `sudo ./arducopter-3_5_4 -A tcp:xxx.xxx.xxx.xxx:14550 -B /dev/ttyS2 -C /dev/ttyS4`

In the Mission Planner remote calibration I am not able to see any green bars, and at this point I am out of ideas and looking for help.

Thank you!!


PS I did bind the remote and the receiver.
PPS Although I have basic knowledge of both, I am not a computer scientist nor an electrical engineer, so instructions in a "for dummies" format are appreciated.
